plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/Connection.java

```diff
@@ -311,6 +311,18 @@ private void clearToken() {
 		setRememberToken(false);
 		saveOrClear(SECURE_STORAGE_TOKEN_KEY, null, false);
 	}
+	
+	public void removeSecureStoreData() {
+		SecureStore store = getSecureStore(getHost(), getUsername());
+		if (store != null) {
+			try {
+				store.removeNode();
+			} catch (SecureStoreException e) {
+				firePropertyChange(SecureStoreException.ID, null, e);
```

Comment:
@fbricon, the consumers are the same as [here](https://github.com/bdshadow/jbosstools-openshift/blob/fa4164c8bed4ea3bee37a4e6057fb0c8022c0536/plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/Connection.java#L221) =)
Should i delete that firePropertyChange too?